framework/console/ErrorHandler.php

```diff
@@ -39,7 +39,7 @@ protected function renderException($exception)
             } elseif (count($alternatives) > 1) {
                 $message .= "\n\nDid you mean one of these?\n    - " . implode("\n    - ", $alternatives);
             }
-        } elseif ($exception instanceof Exception && ($exception instanceof UserException || !YII_DEBUG)) {
+        } elseif ($exception instanceof UserException || !YII_DEBUG) {
```

Comment:
That's not the same condition. Are you absolutely sure `$exception instanceof Exception` is always true?

Reply:
You are right. I should remove `$exception instanceof UserException`.